recipes/r-cmprskcoxmsm/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set version = '0.2.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-cmprskcoxmsm
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/cmprskcoxmsm_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/cmprskcoxmsm/cmprskcoxmsm_{{ version }}.tar.gz
+  sha256: 9366d41e7f23cb753bcd8ef0aa579be7994be9d7d891a19db888818df7b727e0
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-ggplot2
+    - r-sandwich
+    - r-survival
+    - r-twang
+  run:
+    - r-base
+    - r-ggplot2
+    - r-sandwich
+    - r-survival
+    - r-twang
+
+test:
+  commands:
+    - $R -e "library('cmprskcoxmsm')"           # [not win]
+    - "\"%R%\" -e \"library('cmprskcoxmsm')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=cmprskcoxmsm
+  license: GPL-2.0-or-later
+  summary: 'Uses inverse probability weighting methods to estimate treatment effect under marginal
+    structure model for the cause-specific hazard of competing risk events. Estimates
+    also the cumulative incidence function (i.e. risk) of the potential outcomes, and
+    provides inference on risk difference and risk ratio. Reference: Kalbfleisch & Prentice
+    (2002)<doi:10.1002/9781118032985>; Hernan et al (2001)<doi:10.1198/016214501753168154>.'
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
```

Expand license consistent with CRAN:

```suggestion
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3
```